src/api/transaction/types.ts

```diff
@@ -36,7 +36,7 @@ export type Relation = {
 export type Model = {
```

Comment:
What's the point of having `type` and `path` there?

It's a v1 only thing. They'll never populated.